.github/workflows/integration-test-lint.yml

```diff
@@ -32,7 +32,7 @@ on:
         default: 'ubuntu-latest'
 
 concurrency:
-  group: '${{ github.workflow }}-${{ inputs.ref }}'
+  group: '${{ github.workflow }}-${{inputs.repo-to-lint}}-${{ inputs.ref }}-${{inputs.runs-on}}'
```

Comment:
Can we add a comment explaining the intent behind this concurrency? It's not clear looking at it what it does.